content/api/hub/plugin_spin_cron_trigger.md

```diff
@@ -0,0 +1,40 @@
+title = "Spin Cron Trigger"
+template = "render_hub_content_body"
+date = "2022-10-15T00:22:56Z"
+content-type = "text/plain"
+tags = ["trigger", "rust"]
+
+[extra]
+author = "Fermyon"
+type = "hub_document"
+category = "Plugin"
+language = "Rust"
+created_at = "2023-12-05T00:22:56Z"
+last_updated = "2023-12-05T00:22:56Z"
+spin_version = ">v1.0"
```

Comment:
```suggestion
spin_version = “>=2.0"
```

Reply:
@tpmccallum  I don't know if it matters but in a random sample of hub cards we did seem to use a `v` prefix on the version

Reply:
Great spotting @itowlson I guess we should keep with that consistency then @macolso.